webpack/assets/javascripts/react_app/components/about/provider/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Table from '../../common/table';
+import Icon from '../../common/Icon';
```

Comment:
PF-react now has icon component - maybe we should consider using that instead and remove our own?